pkg/sdk/go.mod

```diff
@@ -4,11 +4,14 @@ go 1.16
 
 require (
 	emperror.dev/errors v0.8.0
-	github.com/banzaicloud/istio-client-go v0.0.9
-	github.com/imdario/mergo v0.3.11
+	github.com/banzaicloud/istio-client-go v0.0.10
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/imdario/mergo v0.3.12
 	github.com/jetstack/cert-manager v0.15.2
+	github.com/json-iterator/go v1.1.11 // indirect
+	golang.org/x/net v0.0.0-20210716203947-853a461950ff // indirect
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
 	sigs.k8s.io/controller-runtime v0.8.3
```

Comment:
Can you please bump this to v0.9.2 as well? Thanks.

Reply:
good catch @baluchicken!
Re-run `go get -u=patch ./...` in both root and pkg/sdk go modules